### ISSUE TITLE
Firefox support is broken

### ISSUE THREAD

**Issue by Z-42PO** [anthonyraymond/joal#15]
First of all : well done for this new version !!

Everything works fine, well seems to do, except of the web ui. When I launch Firefox, there is a gray page, the source code shows `<noscript>You need to enable JavaScript to run this app.</noscript>` and the console shows this one : 

```
SecurityError: The operation is insecure.  main.b47f58e5.js:1
	client webstomp.js:871
	value JOALStompClient.js:51:23
	t.connectStomp index.js:25:2
	<anonyme> index.js:9
	t bootstrap c645ea51d4d0d6c06e97:19
	<anonyme> main.b47f58e5.js:11838:17
	t bootstrap c645ea51d4d0d6c06e97:19
	<anonyme> bootstrap c645ea51d4d0d6c06e97:62
	<anonyme> main.b47f58e5.js:1:9
```

What's wrong Doc ?

(yes, obviously javascript is enable ;) )

**Comment by anthonyraymond** [comment]
Woops support for firefox seems to be broken atm i'll take a look at this soon.

**Comment by anthonyraymond** [comment]
@Z-42PO 
I just fixed the bug, the release (v2.0.6) has been published.
If it still doesn't work:
1) Open the JOAL-UI
2) type `localStorage.clear()` into the Firefox console (on JOAL-UI tab).
3) Reload the page with CTRL + SHIFT + R.

If it still does not works it's possible that you have disabled JavaScript on your Firefox, check this out on your side ;)

Thanks for reporting :)

**Comment by kleiton** [comment]
I am using it in firefox and it works perfectly, thank you!
